internal/commands/pull/command_test.go

```diff
@@ -626,6 +628,39 @@ Successfully pulled app down: app
 			}
 		})
 
+		t.Run("should output README path for template", func(t *testing.T) {
+			templateWithReadmeZipPkg, err := zip.OpenReader("testdata/README.md.zip")
+			assert.Nil(t, err)
+			defer templateWithReadmeZipPkg.Close()
+
+			xamarinTemplate := "xamarin.todo"
+			realmClient.CompatibleTemplatesFn = func(groupID, appID string) ([]realm.Template, error) {
+				return []realm.Template{{ID: xamarinTemplate, Name: "Xamarin"}}, nil
+			}
+			realmClient.ClientTemplateFn = func(groupID, appID, templateID string) (*zip.Reader, bool, error) {
+				if templateID == xamarinTemplate {
+					return &templateWithReadmeZipPkg.Reader, true, nil
+				}
+				return nil, false, nil
+			}
+
+			out := new(bytes.Buffer)
+			ui := mock.NewUIWithOptions(mock.UIOptions{AutoConfirm: true}, out)
+
+			profile := mock.NewProfileFromWd(t)
```

Comment:
this test is pulling data/writing to disk, right?  should we be using `NewProfileFromTmpDir` instead?

is that why there's new files for `testdata/template_app/[backend|frontend]/...`?

Reply:
removed new template_app files in favour of using `NewProfileFromTmpDir`